### PR TITLE
Add/edge support

### PR DESCRIPTION
**Pull request by kekkokk**
In march will be released this fix:
[Edge Support](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/12919339/)

This allows Licode to support it.
Note: the Edge publisher side in VP8 still sucks, even in p2p.
So for an "usable" environment we have to wait that Edge fixes its vp8 encoder

[] It needs and includes Unit Tests
[] It includes documentation for these changes in `/doc`.

### CODE REVIEW

File: erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const BaseStack = (specInput) => {
 
   that.pcConfig = {
     iceServers: [],
+    bundlePolicy: 'max-bundle',
```
Comment (jcague) [inline erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js]: have you tested this property with Chrome/Firefox? My only concern is not to break things that are working with the browsers that we currently support.

File: erizo_controller/erizoJS/models/SessionDescription.js [hunk 2]
```diff
@@ -68,7 +68,9 @@ function getMediaInfoFromDescription(info, sdp, mediaType) {
 
   let ice = info.getICECredentials(mediaType);
   if (ice) {
-    media.setICE(new ICEInfo(ice[0], ice[1]));
+    const thisIceInfo = new ICEInfo(ice[0], ice[1]);
+    thisIceInfo.setEndOfCandidates('end-of-candidates');
```
Comment (jcague) [inline erizo_controller/erizoJS/models/SessionDescription.js]: does this break something in Chrome/Firefox?

File: erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js (hunk 1 above)
Comment (kekkokk) [reply erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js]: this breaks p2p firefox, sorry I forgot to update the pr. this must be moved only in EdgeStack, maybe in a separate pr in the future.

File: erizo_controller/erizoJS/models/SessionDescription.js (hunk 2 above)
Comment (kekkokk) [reply erizo_controller/erizoJS/models/SessionDescription.js]: this is bulletproof

Comment (arsenicraghav) [reply erizo_controller/erizoJS/models/SessionDescription.js]: This is breaking things in chrome, local and remote stream turns black after merging this change

**Pull request merged**